app/views/settings/category/_email.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= link_to_function(_("Test email"), "test_mail(this, '#{User.current.mail}', '#{test_mail_user_url(User.current)}')",
```

Comment:
Single quotes are permitted in email addresses, the `User.current.mail` should be escaped or use the ID.